patterns/builder.md

```diff
@@ -2,46 +2,49 @@
 
 ## Description
 
-Construct an object with calls to a builder helper. 
-
+Construct an object with calls to a builder helper.
 
 ## Example
 
 ```rust
-struct Foo {
+#[derive(Debug)]
+pub struct Foo {
     // Lots of complicated fields.
+    bar: String,
 }
 
-struct FooBuilder {
+pub struct Builder {
```

Comment:
Why shouldn't we keep the builder name as `FooBuilder`?

Reply:
On second thought you are right, changed it back ↩️